src/OIDCCryptUtils.purs

```diff
@@ -0,0 +1,48 @@
+module OIDCCryptUtils (RSASIGNTIME(), hashNonce, bindState, unbindState, verifyIdToken) where
```

Comment:
John is criticizing nominative typing there, not saying we shouldn't use newtypes. In fact he thinks we should make extensive use of newtypes when it is the only tool we have ;)